### PR TITLE
Feature/add footer

### PR DESCRIPTION
**Pull request by rebecatschiedel**
|                        Web Dev Path                      |
| :-----------------------------------------------------------: |
| [**38**](https://github.com/MarianaSouza/web-dev-path/issues/38) |

#### What is this change?
A footer structure and style are being added to the project.

#### Were there any complications while making this change?
No.

#### How did you verify this change?
Locally.

#### When should this be merged?
After code review

### CODE REVIEW

File: components/Footer.js [hunk 1]
```diff
@@ -1,21 +1,53 @@
...
+            <li className={footerStyles.footerItem}>
+              <Link href="/blog">
+                <a>Blog</a>
+              </Link>
+            </li>
+            <li className={footerStyles.footerItem}>
+              <Link href="/contact-us">
+                <a>Contact Us</a>
+              </Link>
+            </li>
```
Comment (Schlickmann) [inline components/Footer.js:24]: We could create a constant with the following structure in a utils file or something.

```
const links = [
   { text: 'About Us',  href: '/about-us' },
   { text: 'Blog',  href: '/blog' },
   { text: 'Contact Us',  href: '/contact-us' },
]
```

then here we could do the following to remove some repeated code.

```suggestion
{links.map((link) => (
  <li className={footerStyles.footerItem} key={link.href}>
    <Link href={link.href}>
      <a>{link.text}</a>
    </Link>
  </li>
))}
```

File: components/Footer.js [hunk 2]
```diff
@@ -1,21 +1,53 @@
...
+            </form>
+            <p>*Unsubscribe anytime</p>
+          </div>
+          <div className={footerStyles.socialMedia}>
+            <Link href="#">
+              <img src="" alt="Github" />
+            </Link>
+            <Link href="#">
+              <img src="" alt="Twitter" />
+            </Link>
```
Comment (Schlickmann) [inline components/Footer.js]: Same idea here for the social media links 😄

**Review by Schlickmann — COMMENTED** [review]
Looking good, just left a couple of suggestions here 😃

File: components/Footer.js (hunk 1 above)
Comment (MarianaSouza) [reply components/Footer.js:24]: That's a great idea, Juliani and @rebecatschiedel ! If you feel comfortable about implementing this @rebecatschiedel , pls go ahead.

File: components/Footer.js (hunk 2 above)
Comment (MarianaSouza) [reply components/Footer.js]: Agreed, Juliani! In that same utils file @rebecatschiedel , you could create a `const linksSocial` and map through it after importing the utils file to the component as Juliani mentioned above.

File: pages/index.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import styles from "../styles/Home.module.css";
 
 export default function Home() {
   return (
-    <div className={styles.container}>
+    <div>
```
Comment (MarianaSouza) [inline pages/index.js:6]: Is there any particular reason why you removed the container here, @rebecatschiedel ?

File: styles/Footer.module.css [hunk 4]
```diff
@@ -1,31 +1,131 @@
...
+}
+
+.footerItem:hover,
+.footerItem:focus {
+  color: #666;
+}
+
+.logo {
+  border-radius: 50%;
+  height: 5em;
```
Comment (MarianaSouza) [inline styles/Footer.module.css:34]: Could you pls change height for width: 4.5rem here to make the logo a little bit smaller?

File: public/iconmonstr-twitter-4.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M12 0c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm6.066 9.645c.183 4.04-2.83 8.544-8.164 8.544-1.622 0-3.131-.476-4.402-1.291 1.524.18 3.045-.244 4.252-1.189-1.256-.023-2.317-.854-2.684-1.995.451.086.895.061 1.298-.049-1.381-.278-2.335-1.522-2.304-2.853.388.215.83.344 1.301.359-1.279-.855-1.641-2.544-.889-3.835 1.416 1.738 3.533 2.881 5.92 3.001-.419-1.796.944-3.527 2.799-3.527.825 0 1.572.349 2.096.907.654-.128 1.27-.368 1.824-.697-.215.671-.67 1.233-1.263 1.589.581-.07 1.135-.224 1.649-.453-.384.578-.87 1.084-1.433 1.489z"/></svg>
```
Comment (MarianaSouza) [inline public/iconmonstr-twitter-4.svg:1]: Inside the images folder, could you pls create an svg folder and place the svg files there after renaming them to iconmonstr-github.svg and iconmonstr-twitter.svg (without the numbers), pls?

Also, could you pls delete the logo file that is not in use?

**Review by MarianaSouza — CHANGES_REQUESTED** [review]
Overall the footer structure is great, @rebecatschiedel , thanks so much! Just a few minor changes ;)

File: pages/index.js (hunk 3 above)
Comment (rebecatschiedel) [reply pages/index.js:6]: Not really, I believe I changed after changing the styles but I was supposed to add it back. My mistake!!

File: components/Footer.js (hunk 1 above)
Comment (rebecatschiedel) [reply components/Footer.js:24]: I am on it!!
Thank you @Schlickmann !! Sometimes I forgot I can use logic inside HTML and CSS. 
My navbars will never be the same anymore!! =)

**Comment by rebecatschiedel** [comment]
Is the favicon working for anyone? It worked for me, at first, but now it is gone.

**Review by MarianaSouza — APPROVED** [review]
Way to go, @rebecatschiedel !

**Pull request merged**